pkg/harvester/models/harvesterhci.io.virtualmachineimage.js

```diff
@@ -10,7 +10,7 @@ import { formatSi } from '@shell/utils/units';
 import { ucFirst } from '@shell/utils/string';
 import { stateDisplay, colorForState } from '@shell/plugins/dashboard-store/resource-class';
 import { _CLONE } from '@shell/config/query-params';
-import { isReady } from '@shell/machine-config/harvester';
+import { isReady } from '../../harvester-manager/machine-config/harvester';
```

Comment:
This is something I should have caught when making the harvester-manager extension, sorry about that. Does this mean we need a patch for the harvester-1.0.3 extension to be compatible with Rancher 2.7.next1? Either way it might actually be easier to maintain duplicate code of `isReady` here than to create a dependency on the harvester-manager extension in the harvester extension.

Reply:
The harvester-1.0.3 extension has been built so the `isReady` have existed in the Harvester plugin assets. It won't cause compatible issues. 

And I will fix this line in the Harvester repo. Because this is about Harvester plugin.

Reply:
Ok, thank you for clarifying :)